lib/matplotlib/backends/backend_ps.py

```diff
@@ -422,17 +408,15 @@ def option_image_nocomposite(self):
         # docstring inherited
         return not rcParams['image.composite_image']
 
-    def _get_image_h_w_bits_command(self, im):
-        h, w, bits = self._rgb(im)
-        imagecmd = "false 3 colorimage"
-
-        return h, w, bits, imagecmd
-
     def draw_image(self, gc, x, y, im, transform=None):
         # docstring inherited
 
-        h, w, bits, imagecmd = self._get_image_h_w_bits_command(im)
-        hexlines = b'\n'.join(self._hex_lines(bits)).decode('ascii')
+        h, w = im.shape[:2]
+        imagecmd = "false 3 colorimage"
+        data = im[::-1, :, :3]
```

Comment:
maybe `rgb_data` instead of `data` to be a bit more telling?

Reply:
It's in draw_image; what else can it be?
If anything the interesting part is that it's upside down, not that it's rgb.

Reply:
```suggestion
        data = im[::-1, :, :3]  # vertically flipped rgb values
```
might help when someone reads the code.